src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -264,48 +264,34 @@ private async ValueTask<ReSharperStatus> IsReSharperRunningAsync(ReSharperStatus
 
             await EnsureOleCommandTargetAsync().ConfigureAwait(false);
 
-            var cmds_suspend = new OLECMD[1];
-            cmds_suspend[0].cmdID = SuspendId;
-            cmds_suspend[0].cmdf = 0;
-
-            var cmds_resume = new OLECMD[1];
-            cmds_resume[0].cmdID = ResumeId;
-            cmds_resume[0].cmdf = 0;
-
-            // poll until either suspend or resume button is available, or until operation is canceled
+           // poll until either suspend or resume botton is available, or until operation is canceled
             while (true)
             {
                 cancellationToken.ThrowIfCancellationRequested();
 
-                var hr_suspend = await QueryStatusAsync(cmds_suspend).ConfigureAwait(false);
+                var suspendFlag = await QueryStatusAsync(SuspendId).ConfigureAwait(false);
 
                 // In the case of an error when attempting to get the status, pretend that ReSharper isn't enabled. We also
                 // shut down monitoring so we don't keep hitting this.
-                if (ErrorHandler.Failed(hr_suspend))
```

Comment:
Please continue to use the HResult comparison and not use magic numbers.

Reply:
➡️ I requested that the HR check be contained within the helper, so it will not be present at this location. This location will operate on `OLECMDF`. The magic number 0 will be required because this enumeration does not have a named `None` value; the value 0 is treated as equivalent to `None`.